packs/drupal.js

```diff
@@ -0,0 +1,44 @@
+  /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using webp in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  uses_webp_images: 'Consider using a module such as [WebP for Drupal](https://www.drupal.org/project/webp) that will automatically generate a WebP version of your uploaded images to optimize loading times.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by lazy loading images that are initially offscreen in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  offscreen_images: 'Install a Drupal module that can leverage JavaScript or [Intersection Observer API](https://developers.google.com/web/updates/2016/04/intersectionobserver) lazy loading of images such as [Blazy](https://www.drupal.org/project/blazy). Such modules provide the ability to defer any offscreen images to improve performance.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site loading performance by reducing the total bytes delivered by their page in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  total_byte_weight: 'Consider using [Responsive Image Styles](https://www.drupal.org/docs/8/mobile-guide/responsive-images-in-drupal-8) to reduce the size of images loaded in your page. If you are using Views to show multiple content items in a page, consider implementing pagination to limit the number of content items shown on a given page.',
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by reducing the amount of render blocking resources present on their page, in the context of the Drupal CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  render_blocking_resources: 'Consider using the [Advanced CSS/JS Aggregation](https://www.drupal.org/project/advagg) module to inline critical CSS or potentially load assets asynchronously via JavaScript. Beware that optimizations provided by this module may break your site, so you will likely need to make code changes.',
```

Comment:
Instead of pointing a single module here as well, could we point to a list (search page, etc...)?

Reply:
The only module that can help in this function (other than manually rewriting a theme to remove rendering blocking) is the Advanced CSS/JS Aggregation. 
Nevertheless, I have rewrote the recommendation to hint to the module instead of basing the fix on it.

It now reads as:

> Consider using a module to inline critical CSS and JavaScipt, or potentially load assets asynchronously via JavaScript such as the [Advanced CSS/JS Aggregation](https://www.drupal.org/project/advagg) module. Beware that optimizations provided by this module may break your site, so you will likely need to make code changes.